lib/spoom/cli/srb/sigs.rb

```diff
@@ -53,6 +53,118 @@ def strip(*paths)
           say("Stripped signatures from `#{transformed_files}` file#{transformed_files == 1 ? "" : "s"}.")
         end
 
+        # Extract signatures from gem files and save them to the output file
+        #
+        # This command will use Tapioca to generate a `.rbi` file that contains the signatures of all the files listed
+        # in the gemspec.
+        desc "export", "Export gem files signatures"
+        option :gemspec, type: :string, desc: "Path to the gemspec file", optional: true, default: nil
+        option :check_sync, type: :boolean, desc: "Check the generated RBI is up to date", default: false
+        def export(output_path = nil)
+          gemspec = options[:gemspec]
+
+          unless gemspec
+            say("Locating gemspec file...")
```

Comment:
Not a big deal but we could clean up the nested `unless` if we're okay with getting rid of this say
`gemspec = options[:gemspec] || Dir.glob("*.gemspec").first`